client/add/box.view.html

```diff
@@ -0,0 +1,35 @@
+<md-dialog aria-label="Add new box" ng-cloak>
+  <form>
+    <md-toolbar>
+      <div class="md-toolbar-tools">
+        <h2>Add new box</h2>
+        <span flex></span>
+        <md-button class="md-icon-button" ng-click="cancel()">
+          <md-icon md-svg-src="img/icons/ic_close_24px.svg" aria-label="Close dialog"></md-icon>
+        </md-button>
+      </div>
+    </md-toolbar>
+    <md-dialog-content>
+      <div class="md-dialog-content" layout="column">
+        <md-input-container class="md-block">
+            <label>Name</label>
```

Comment:
Yeah, I probably need some more nice ux around this. I was just so excited I finally fixed the issues with angular material.